test/data/tsang-falloff.yaml

```diff
@@ -0,0 +1,224 @@
+generator: cti2yaml
+cantera-version: 2.6.0a2
+date: Mon, 30 Aug 2021 16:33:51 +0300
+input-files: [Lucassen-2011.cti]
```

Comment:
Instead of the auto-generated header block, it may be more informative to add
```yaml
description: |-
  some_description_of_what_this_file_is_used_for
```
with a suitable description replacing `some_description_of_what_this_file_is_used_for`.  If `Lucassen-2011` is relevant, provide context.

Reply:
Added description with DOI